### ISSUE TITLE
Documentation: `tflocal apply` fails - AWS region needs to be explicitly set in `main.tf`

### ISSUE THREAD

**Issue by techpink** [localstack/terraform-local#5]
terraform-local 0.4
localstack-client 1.39
terraform 1.3.1
localstack 1.2.0

The [terraform integration doc](https://docs.localstack.cloud/integrations/terraform/) suggests testing `tflocal` with the minimal `main.tf` file below:
```
resource "aws_s3_bucket" "test-bucket" {
  bucket = "my-bucket"
}
```

And then running `tflocal init` followed by `tflocal apply`

`tflocal apply` fails with

```
% tflocal apply
╷
│ Error: Invalid AWS Region:
│
│   with provider["registry.terraform.io/hashicorp/aws"],
│   on localstack_providers_override.tf line 2, in provider "aws":
│    2: provider "aws" {
│
╵
```
This can be fixed by adding an `aws` provider block in the `main.tf` that sets the region, like this:
```
provider "aws" {
  region     = "us-east-1"
}

resource "aws_s3_bucket" "test-bucket" {
  bucket = "my-bucket"
}
```

```
% tflocal apply

Terraform used the selected providers to generate the following execution plan. Resource actions are indicated with the following symbols:
  + create

Terraform will perform the following actions:

  # aws_s3_bucket.test-bucket will be created
  + resource "aws_s3_bucket" "test-bucket" {

...

      + bucket                      = "my-bucket"

...

    }

Plan: 1 to add, 0 to change, 0 to destroy.

Do you want to perform these actions?
  Terraform will perform the actions described above.
  Only 'yes' will be accepted to approve.

  Enter a value: yes

aws_s3_bucket.test-bucket: Creating...
aws_s3_bucket.test-bucket: Creation complete after 1s [id=my-bucket]

Apply complete! Resources: 1 added, 0 changed, 0 destroyed.
```

**Comment by whummer** [comment]
Thanks for reporting. This should be fixed in version `0.5` - can you please install the latest version and give it another try? Please let us know if the problem persists or if there any other issues. Thanks!

**Comment by maher-salla** [comment]
the new version 0.5 is not working at all.
I tried the following two snippets:
```hcl
provider "aws" {
  region                      = "eu-central-1"
  access_key                  = "0"
  secret_key                  = "0"
  skip_credentials_validation = true
  skip_requesting_account_id  = true

  endpoints {
    ec2 = "http://localhost:4566"
  }
}

resource "aws_instance" "web" {
  ami                    = "ami-25488752"
  instance_type          = "t2.micro"
  vpc_security_group_ids = ["${aws_security_group.web.id}"]
  user_data              = file("template/user_data.sh")
}

resource "aws_security_group" "web" {
  ingress {
    from_port   = 80
    to_port     = 80
    protocol    = "tcp"
    cidr_blocks = ["0.0.0.0/0"]
  }
}

```

```hcl
resource "aws_instance" "web" {
  ami                    = "ami-25488752"
  instance_type          = "t2.micro"
  vpc_security_group_ids = ["${aws_security_group.web.id}"]
  user_data              = file("template/user_data.sh")
}

resource "aws_security_group" "web" {
  ingress {
    from_port   = 80
    to_port     = 80
    protocol    = "tcp"
    cidr_blocks = ["0.0.0.0/0"]
  }
}

```

I downgraded to 0.4 and it worked
`pip3 install 'terraform-local==0.4' --force-reinstall
